src/common/base/base.alert.scss

```diff
@@ -0,0 +1,4 @@
+.alert {
+    text-align: center;
+    color: red;
```

Comment:
@PJSosnowski -- let's not use `red` -- please use a Sass variable so we can redefine this value later -- I'd call this color $errorColor (or something)

You should [define variables over here](https://github.com/motech/motech-ui/blob/master/src/app/app.variables.scss)